docs/configuration/secure.md

```diff
@@ -0,0 +1,571 @@
+chown [www-data:www-data|www_ilias_1:www_ilias_1] /var/www/ilias_log/ilias_errors
+
+chmod [2]775 /var/www/ilias/data
+chmod [2]775 /var/www/ilias_data
+chmod [2]775 /var/www/ilias_log
+chmod [2]775 /var/www/ilias_log/ilias_errors
+```
+
+After the installation of ILIAS is finished, you SHOULD also revoke write permission for the file ```ilias.ini.php``` (e.g. ```/var/www/ilias/ilias.ini.php```).
+note: for changing base setting via ILIAS setup, you need to grant write permission for the file ```ilias.ini.php``` again.
+
+## Place data directory outside of the web root
+
+It is highly RECOMMENDED to place your data directory outside of the web server docroot, as pointed out by the ILIAS Installation Wizard.
+
+## Isolate Docroot
+You MAY use openbasedir-restriction to avoid malicious software to directory-traverse out of your docroot-directory. This is very important if there are other websites on the same host.
+
+Apache2:
+
+    php_admin_value open_basedir ./:%docroot_dir%:/usr/share/php/:/var/www/lib/:/tmp
+
+Nginx:
+
+    fastcgi_param PHP_VALUE open_basedir="./:%docroot_dir%:/usr/share/php/:/var/www/lib/:/tmp";
+**Hint:** This option will be applied to the PHP-FPM process. If there are multiple websites on your webserver you have to define a single PHP-FPM-pool for each website. Otherwise these other homepages won' t be accessible anymore.
+
+## OS user handling security
+
+If you use PHP-FPM (FastCGI Process Manager), you can increase security by running the PHP-FPM processes as an specific instead ```www-data``` / ```wwwrun``` (depends on linux distribution).
+
+Snippet from a PHP-FPM pool definition:
+```
+; Unix user/group of processes
+; Note: The user is mandatory. If the group is not set, the default user's group
+;       will be used.
+user = www_ilias_1
+group = www_ilias_1
+```
+
+**note:**  
+NGINX and also apache2 is can only run with one user (no multi user multi process model).  
+So it is necessary to put all the "PHP-FPM"-users in the primary group of the webserver user.
+
+## Major security improvement: use HTTPS
+
+You can get a trusted, free SSL certificate at https://letsencrypt.org.
+Or use a SSL certificate from a commercial certificate authority.
+
+### Redirect all unencrypted traffic to HTTPS
+
+To redirect all HTTP traffic to HTTPS you SHOULD issue a permanent redirect using the 301 status code:
+
+#### Apache2
+
+```
+<VirtualHost *:80>
+   ServerName yourservername.org
+   Redirect permanent / https://yourservername.org/
+</VirtualHost>
+```
+
+#### NGINX
+
+```
+server {
+		listen   *:80;
+		listen   [::]:80;
+		server_name yourservername.org;
+
+		location / {
+				return 301 https://yourservername.org$request_uri;				
+		}
+```
+
+### Proper SSL configuration
+
+The default SSL settings (e.g. ciphers & ssl protocol version ) used by web servers are often not state-of-the-art, so you SHOULD consider to choose your own settings.
+
+**Suggestion of modern settings for SSL configuration:**
```

Comment:
Instead of a static recommendation we just might should advice the user to use https://ssl-config.mozilla.org to get a more suited config for his environment.

Reply:
Absolutely. These recommendations will be out-of-date in no time.

Reply:
There is a link to "Mozilla SSL Configuration Generator" in the following text.

We add a sentence that is necessary to revise this setting.  

The workshop discus that they have to be a suggested configuration for new admins.
We promise to hold this settings modern in future in this document.